src/Microsoft.NET.Sdk.Aspire/WorkloadManifest.in.json

```diff
@@ -7,7 +7,8 @@
         "Aspire.Hosting.Sdk",
         "Aspire.ProjectTemplates",
         "Aspire.Hosting.Orchestration",
-        "Aspire.Hosting"
+        "Aspire.Hosting",
+        "Aspire.Dashboard"
```

Comment:
We should consider using a different package ID (say `Aspire.Dashboard.Sdk`) to differentiate the new workload pack nuget from the original PackageReference dashboard nuget.